vochain/apptest.go

```diff
@@ -115,8 +123,37 @@ func (app *BaseApplication) AdvanceTestBlock() {
 	}
 	// The next block begins 50ms later
 	newHeight := app.testMockBlockStore.EndBlock()
-	time.Sleep(time.Millisecond * 50)
+	time.Sleep(time.Microsecond * 50)
 	nextStartTime := time.Now()
 	app.testMockBlockStore.NewBlock(newHeight)
 	app.beginBlock(nextStartTime, uint32(newHeight))
 }
+
+// AdvanceTestBlocksUntilHeight is equivalent to AdvanceTestBlock,
+// but loops until reaching height n
+func (app *BaseApplication) AdvanceTestBlocksUntilHeight(n uint32) {
```

Comment:
why this function does not use AdvanceTestBlock ?

Reply:
i was unsure about reducing time.Sleep of AdvanceTestBlock would break things. fixed.